phoenicis-repository/src/test/java/org/phoenicis/repository/repositoryTypes/CachedRepositoryTest.java

```diff
@@ -38,8 +38,8 @@ public void testFetchInstallableApplications() throws Exception {
                         .thenReturn(
                                 new RepositoryDTO.Builder()
                                         .withCategories(
-                                                Arrays.asList(new CategoryDTO.Builder().withName("Category 1").build(),
-                                                        new CategoryDTO.Builder().withName("Category 2").build()))
```

Comment:
I think I got your point. However, I'm still unsure how this should be implemented. A new class `TranslatableApplicationDTO` which is derived from `ApplicationDTO` and implements `Translatable`? And then during the fetch create first an `ApplicationDTO` and then a `TranslatableApplicationDTO` from it and call `tr(TranslatableApplicationDTO)`?